application/src/main/resources/ecc.yml

```diff
@@ -261,3 +277,28 @@ rest_server:
   ##
   host: localhost
   port: 8080
+
+  lock_factory:
+    cas:
+      ##
+      ## The keyspace used for the CAS lock factory tables.
+      ##
+      keyspace: ecchronos
+      ##
+      ## The number of seconds until the lock failure cache expires.
+      ## If an attempt to secure a lock is unsuccessful,
+      ## all subsequent attempts will be failed until
+      ## the cache expiration time is reached.
+      ##
+      cache_expiry_time_in_seconds: 30
+      ##
+      ## Allow to override consistency level for LWT (lightweight transactions). Possible values are:
+      ## "DEFAULT" - Use consistency level based on remoteRouting.
+      ## "SERIAL" - Use SERIAL consistency for LWT regardless of remoteRouting.
+      ## "LOCAL" - Use LOCAL_SERIAL consistency for LWT regardless of remoteRouting.
+      ##
+      ## if you use remoteRouting: false and LOCAL then all locks will be taken locally
+      ## in DC. I.e There's a risk that multiple nodes in different datacenters will be able to lock the
+      ## same nodes causing multiple repairs on the same range/node at the same time.
+      ##
+      consistencySerial: "DEFAULT"
```

Comment:
How is it working without remoteRouting???

Reply:
now this would select consistency level Serila

Reply:
I think if should work in the same way, but instead of checking for remoting routing, if should check if agent.type is datacenterAware

Reply:
done!